.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @guardian/developer-experience-stream
```

Comment:
I'm not sure about this. Abstractly, this repository describes the list of repositories a team owns/is interested in. IIUC a CODEOWNERS file will automatically assign the listed team as a reviewer of a pull request. Should `@guardian/developer-experience-stream` be this gatekeeper?

I wonder if should add a pull request template to this repository too, to signal that whilst `@guardian/developer-experience-stream` might get auto-assigned as a reviewer, an approval from them is not mandatory. See [`guardian/riffraff-platform`](https://github.com/guardian/riffraff-platform/blob/main/.github/PULL_REQUEST_TEMPLATE.md?plain=1) as an example.

Alternatively, the [department](https://github.com/orgs/guardian/teams/product-engineering-department) could be a CODEOWNER?

Reply:
My understanding is that how the CODEOWNERS file is used depends on the branch protection rules.  So it can either auto-assign reviewers or be a mandatory approver or have no effect at all.  So maybe it should be set to have no effect on PRs but still stand to show ownership of the repo for anyone who's browsing.

Reply:
Actually that's not true - they get auto-assigned as a reviewer whatever you do.  But could set `guardian/developer-experience-stream` as owner of just the CODEOWNERS file to indicate repo ownership without the noise